website/pages/docs/concepts/seal.mdx

```diff
@@ -84,67 +84,124 @@ to decrypt the master key at startup and unseal automatically.
+keys.
+
+``` $ vault operator unseal -migrate ```
+
+#### Migration From Auto Unseal to Shamir
 
 ~> **NOTE**: Migration to Shamir seal is not currently supported when using
 Vault Enterprise. We plan to support this officially in a future release.
 
-To migrate from Auto Unseal to Shamir keys, take your server cluster offline and update
-the [seal configuration](/docs/configuration/seal) and add `disabled = "true"`
-to the seal block. This allows the migration to use this information to decrypt the key
-but will not unseal Vault. When you bring your server back up, run the unseal process
-with the `-migrate` flag and use the Recovery Keys to perform the migration. All unseal
-commands must specify the `-migrate` flag. Once the required threshold of recovery keys
-are entered, the recovery keys will be migrated to be used as unseal keys.
+To migrate from Auto Unseal to Shamir keys, take your server cluster offline and
+update the [seal configuration](/docs/configuration/seal) and add `disabled =
+"true"` to the seal block. This allows the migration to use this information to
+decrypt the key but will not unseal Vault. When you bring your server back up,
+run the unseal process with the `-migrate` flag and use the Recovery Keys to
+perform the migration. All unseal commands must specify the `-migrate` flag.
+Once the required threshold of recovery keys are entered, the recovery keys will
+be migrated to be used as unseal keys.
 
-### Migration From Auto Unseal to Auto Unseal
+#### Migration From Auto Unseal to Auto Unseal
 
 ~> **NOTE**: Migration between same Auto Unseal types is not currently
 supported. We plan to support this officially in a future release.
 
-To migrate from Auto Unseal to a different Auto Unseal configuration, take your server
-cluster offline and update the existing [seal configuration](/docs/configuration/seal)
-and add `disabled = "true"` to the seal block.  Then add another seal block to describe
-the new seal.
+To migrate from Auto Unseal to a different Auto Unseal configuration, take your
+server cluster offline and update the existing [seal
+configuration](/docs/configuration/seal) and add `disabled = "true"` to the seal
+block.  Then add another seal block to describe the new seal.
 
-When you bring your server back up, run the unseal process with the `-migrate` flag and
-use the Recovery Keys to perform the migration. All unseal commands must specify
-the `-migrate` flag. Once the required threshold of recovery keys are entered,
-the recovery keys will be kept and used as recovery keys in the new seal.
+When you bring your server back up, run the unseal process with the `-migrate`
+flag and use the Recovery Keys to perform the migration. All unseal commands
+must specify the `-migrate` flag. Once the required threshold of recovery keys
+are entered, the recovery keys will be kept and used as recovery keys in the new
+seal.
 
-### Migration with Integrated Storage
+#### Migration with Integrated Storage
```

Comment:
Given the header nesting, I read this as still Integrated Storage pre 1.4, and that integrated storage post 1.4 would use the other set of instructions - is that accurate?

Reply:
Yes. That is correct.